### PR TITLE
cgobj, cgsched: Fix indentation

### PR DESCRIPTION
**Pull request by Geod24**
Just a boring collection of indentation fixes, no actual code change.
Trivially reviewed if you disable whitespace changes in the diff viewer.

### CODE REVIEW

**Review by JinShil — APPROVED** [review]

File: src/dmd/backend/cgsched.d [hunk 1]
```diff
@@ -1759,18 +1765,19 @@ else
...
-                        c.IFL1 = FLconst;
-                        c.IEV1.Vpointer = 0;
-                        irm |= 0x80;
-                        break;
-                    case 1:
-                        c.IEV1.Vpointer = cast(byte) c.IEV1.Vpointer;
-                        irm = modregrm(2,0,rm);
-                        break;
+                {
+                case 0:
```
Comment (WalterBright) [inline src/dmd/backend/cgsched.d]: I never liked this style of indentation. Please don't.

**Review by WalterBright — CHANGES_REQUESTED** [review]
The indentation changes are fine, except for the switch statement unindent.

The * changes are just churn. I tend to fix them when I'm changing the same line anyway. Going through the whole backend to "fix" them accomplishes nothing.

Same for adding spaces between arguments.

File: src/dmd/backend/cgsched.d (hunk 1 above)
Comment (Geod24) [reply src/dmd/backend/cgsched.d]: Fixed

**Comment by Geod24** [comment]
> The * changes are just churn. I tend to fix them when I'm changing the same line anyway. Going through the whole backend to "fix" them accomplishes nothing.

Having commits that mix indentation change and actual code change are a distraction to the reviewer. They also make revert harder. I'd rather have one big commit which only changes indentation and can be trivially reviewed than one that mixes style fixes and behavior changes. But I know that's something you disagree with.

But anyway, I  have no intent to give the full backend this treatment at the moment.

**Comment by WalterBright** [comment]
> Having commits that mix indentation change and actual code change are a distraction to the reviewer. They also make revert harder.

Of course you know I say that all the time. I also say use best judgement. Changing a line or two in the code you're changing anyway is not a problem. Having one line of behavior change surrounded by 50 lines of reformatting **is** a problem.

Globally changing thousands of lines of `T*` to `T *` is churn and of no particular benefit. Ditto for ten thousand lines of `foo(a,b)` to `foo(a, b)`. You're wasting your time and everyone else who looks at git blame and `Geod24` is on most every line.

Changing the indent is worthwhile, though. I did most of that already when I changed every line anyway when converting it from C++ to D, since the git blame was already hosed.

> But anyway, I have no intent to give the full backend this treatment at the moment.

But someone else will later and point to this PR as justification.

You're an excellent developer, may I suggest work that will add significant value, like adding unit tests like what I did for OutBuffer https://github.com/dlang/dmd/pull/10269 to fill in missing code coverage. Reformatting code is not addressing the technical debt problems.

**Comment by RazvanN7** [comment]
So, what are we going  to do with this?

**Comment by Geod24** [comment]
Well, I'd very much like it merged since this is fixing some non-contentious style issues.

File: src/dmd/backend/cgobj.d [hunk 2]
```diff
@@ -1568,39 +1590,40 @@ private void objheader(char *csegname)
   }
   objrecord(COMENT,comment.ptr,comment.sizeof);
 
-    // Send out comment indicating we're using extensions to .OBJ format
```
Comment (thewilsonator) [inline src/dmd/backend/cgobj.d]: why change 4 spaces to two? Ditto throughout.

Comment (Geod24) [reply src/dmd/backend/cgobj.d]: Hum right, I didn't realize most of the function was misindented.

Comment (Geod24) [reply src/dmd/backend/cgobj.d]: Fixed

**Review by thewilsonator — APPROVED** [review]

**Comment by Geod24** [comment]
So we just let this rot?

**Comment by WalterBright** [comment]
> So we just let this rot?

Please implement the changes I requested and I'll pull it.

**Comment by Geod24** [comment]
> Please implement the changes I requested and I'll pull it.

Ah I misunderstood your last message. Removed the `(a,b)` and `T *` changes, unless they were on a line that was being touched anyway (e.g. indent fixes).

**Comment by Geod24** [comment]
Ping @WalterBright

**Review by WalterBright — APPROVED** [review]

**Pull request merged**